### ISSUE TITLE
why did you use normalization = False in librosa.write in separate.py ?

### ISSUE THREAD

**Issue by SashankMLAI** [kaituoxu/Conv-TasNet#9]
when I put normalization = True the output is clear without any disturbance. did you get estimate_source values between -1 to 1 using normalization = False?
As this is done in time domain, our output range is -1 to 1 but, when we use Relu the output can be more than 1 right? Please help!

**Comment by kaituoxu** [comment]
What did you mean normalization=False?
This line? https://github.com/kaituoxu/Conv-TasNet/blob/master/src/separate.py#L54 ?

**Comment by SashankMLAI** [comment]
Yes! I used your pretrained model, Only when I put norm = true in librosa.write, I am getting the output, otherwise, the amplitude is more than 1 or less than -1 for every sample and file becomes non playable.     

If we put normalization true, for a real time case, if we are processing every 10ms, and if we are using norm = true for writing the output, there is a chance that the entire 10ms is noise and it gets boosted aswell. 

What I observed is, output is between -1 to 1 for the trained wav files, but not the same with unseen data. Any thoughts on this? correct me if I am wrong

**Comment by kaituoxu** [comment]
In this librosa doc (https://librosa.github.io/librosa/0.4.3/generated/librosa.output.write_wav.html?highlight=write#librosa.output.write_wav) , norm=true is default, maybe different version of librosa has different default norm.

**Comment by SashankMLAI** [comment]
True, got it. Thanks. 
Do you have any advice to make it work in real-time? If we are processing every 10ms, and if we are using norm = true for writing the output in Librosa, there is a chance that the entire 10ms is noise and it gets boosted aswell.

**Comment by kaituoxu** [comment]
Noisy result maybe because train/test mismatch.

The pre-trained model I provided is trained in non-causal way (i.e. use history and future information) and use audio segments longer than 10ms (2s? 4s? I don't remember..), so if you process 10ms, maybe it is too short and don't have enough future information.

You can train a causal model (i.e. only use history information), maybe you can get better result.
